recipes/protgraph/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "protgraph" %}
+{% set version = "0.0.2b3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c13e3bc3e89ca95977a1f42374a4223082cd7a0c907ac0d61872cf038f336080
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - protgraph=protgraph.protgraph:main
+
+requirements:
+  host:
+    - biopython
+    - mysql-connector-python
+    - networkx
+    - pip
+    - python >=3.6
+    - tqdm
+    - aenum
+    - isodate
+    - wcwidth
+    - importlib_metadata
+    # These packages are not available in conda!
```

Comment:
These will need to be added to conda-forge

Reply:
How should i proceed? I found some of the packages in `conda-forge`, but the packages: `redisgraph` as well as `gremlinpython` are still not available in conda. Both of them are not bio-related (graph tools). 

Is there an option to install them from PyPI using the recipe itself instead of the `build.sh`?

Reply:
You'll need to add them to conda-forge by making a PR here: https://github.com/conda-forge/staged-recipes